config/locales/en.yml

```
@@ -75,8 +75,12 @@ en:
             invalid: There was a problem updating this answer
             success: Answer successfully updated
         census:
+          census_type:
+            external: External (upload a CSV file)
+            internal: Internal (all registered participants)
```

As you can segment by different permissions, it's better to drop the `all`
```suggestion
            internal: Internal (registered participants)
```